src/renderer/redux/reducers/settings.js

```diff
@@ -19,6 +19,7 @@ const defaultState = {
     showUnavailable: getLocalStorageSetting(SETTINGS.SHOW_UNAVAILABLE, true),
     welcome_acknowledged: getLocalStorageSetting(SETTINGS.NEW_USER_ACKNOWLEDGED, false),
     email_collection_acknowledged: getLocalStorageSetting(SETTINGS.EMAIL_COLLECTION_ACKNOWLEDGED),
+    invite_acknowledged: getLocalStorageSetting(SETTINGS.INVITE_ACKNOWLEDGED, false),
```

Comment:
All of these first run "settings" probably shouldn't live here. Not sure how tricky it would be to migrate them just to regular values saved by redux-persist. Then we can get rid of `getLocalStorageSetting`.

Works fine for now though, but something to keep in mind.

Reply:
kill it with :fire: